### DISCUSSION TITLE
automatically set the right theme according to prefers-color-scheme

### DISCUSSION THREAD

**Discussion by tleydxdy** [thuliteio/doks#445] [Ideas]
https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme

currently the theme only work when JS is enabled it seems

**Comment by h-enk** [comment]
Will be implemented soon (including no [FART](https://css-tricks.com/flash-of-inaccurate-color-theme-fart/)) — implemented yesterday on [getdoks.org](https://getdoks.org/)

**Reply by tleydxdy** [reply]
I tried out getdoks.org, still doesn't seem to be working. (without JS at least)
